src/ert/validation/realizations_in_ensemble_argument.py

```diff
@@ -0,0 +1,51 @@
+from typing import TYPE_CHECKING, Optional
+
+from .range_string_argument import RangeStringArgument
+from .rangestring import rangestring_to_list
+from .validation_status import ValidationStatus
+
+if TYPE_CHECKING:
+    from ert.storage import Ensemble
+
+
+class RealizationsInEnsembleArgument(RangeStringArgument):
+    UNINITIALIZED_REALIZATIONS_SPECIFIED = (
+        "The specified realization(s) %s are not found in selected ensemble."
```

Comment:
Can we append the `ensemble.name`?; ie.  "The specified realization(s) %s are not found in selected ensemble %s."?

Reply:
Ignore :)